docs/editors-guide/rare-disease-subset.md

```diff
@@ -0,0 +1,56 @@
+# Rare Disease Subset in Mondo
+
+A rare disease is a disease that affects fewer than 200,000 people in the United States. However, this definition is not universal and varies between rare disease sources, geographical locations, etc. Consequently, rare disease criteria also vary between sources, and one might have difficulties reconciling sources and creating a list for rare diseases. Mondo recognizes these impediments and differences of view. This document explains how Mondo handles rare diseases, how one could get a rare disease list, and the efforts made to bring the community together.
+
+#### What are rare diseases in Mondo?
+There are 3 sources for rare diseases in Mondo:
+1. rare disease authorities
+2. manual curation based on literature
+3. ontology inference
+
+**1. rare disease authorities**
+We rely on rare disease authorities (see table below) to determine diseases recognized as “rare”. These rare diseases are determined by the rare disease authorities according to their own criteria. When a disease is considered “rare” according to a source, the corresponding Mondo term is annotated with an `in_subset` annotation expressing that this disease is considered rare according to this source: `X_rare`  (where X is the rare disease authority). These rare disease subset annotations are maintained automatically via rare disease lists from the authorities.  
+
+Example: 'calciphylaxis' (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.
+![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)
+
+**2. manual curation**
+In some occasional cases, some diseases are reported in the literature as 'rare', though they are not reported in the rare disease authorities' list. In these cases, we record the disease as rare by manually adding the "mondo_rare" subset annotation to the term. These manually curated annotations must be supported by the literature. This new rare disease is reported to the rare disease authorities. 
+
+**3. ontology inference**
+Ontological parents pass their characteristics (here: axioms) to their children. We leverage this ontological property for rare diseases: ontological children of a term that is considered "rare" according to the rules above are getting the "inferred_rare" subset annotation.
+
+
+#### The Mondo rare disease subset includes ALL rare diseases according to ALL rare disease sources.
+The Mondo rare disease subset represents the UNION of rare diseases according to all authority sources, manual curation, and ontology inference. It should be noted that most of the diseases in this subset will be viewed as “rare” by _most_ authorities, but some will be viewed as “rare” by _only some_ authorities. 
```

Comment:
Have you checked this with the subset report query? I seem to remember there were huge differences. We could also add an overlap plot here (venn diagram) to illustrate the overlap. You can make a separate issue if you would like that (not a blocker here).

Reply:
I haven't compared the different subsets and I agree having a venn diagram would help. Let's talk about how to do this, because it is not necessarily straight forward. I will make a separate issue.